### PR TITLE
Fix vtcompose command incorrect regex

### PR DESCRIPTION
**Pull request by DAlperin**
## Description
The vtcompose utility in `examples/compose/` had an incorrect regex which would make it panic on the provided example schema files.

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported
-   [x] Tests were added or are **not required**
-   [x] Did the new or modified tests pass consistently locally and on the CI
-   [x] Documentation was added or is not required

### CODE REVIEW

**Review by frouioui — APPROVED** [review]
Thanks for your first contribution @DAlperin, this change looks good to me.

I see the item `Did the new or modified tests pass consistently locally and on the CI`, from the PR's checklist is not ticked off, did the compose example passed locally several time on your machine?

**Comment by DAlperin** [comment]
@frouioui oh yep sorry! Didn't think it counted. I tested it locally and it did what I expected (didn't panic, generated working configs). There are some other problems with the docker-compose example on ARM macs but those are out of scope of this PR.

**Comment by frouioui** [comment]
> 

Awesome thank you @DAlperin. Just waiting for a second review and we're good to go.

**Review by mattlord — APPROVED** [review]
I had one suggestion. Let me know what you think. I'll approve for now and we can merge this once we hear back.

File: examples/compose/vtcompose/vtcompose.go
Comment (DAlperin) [reply examples/compose/vtcompose/vtcompose.go]: Good call

File: examples/compose/vtcompose/vtcompose.go
Comment (mattlord) [inline examples/compose/vtcompose/vtcompose.go:320]: Gah! I'm sorry... I just realized that we should be requiring *at least* one space in a few spots there so that we don't also match e.g. `CREATETABLE`:
```suggestion
	r, _ := regexp.Compile("(?i)CREATE[[:space:]]+TABLE[[:space:]]+(IF NOT EXISTS)?[[:space:]]+([a-z_-]*)[[:space:]]*\\(")
```
https://go.dev/play/p/nxjMA7SiTsW

I'm sorry that I caused yet another round trip on that. 😞

**Pull request closed without merging**
